static/css/crateio-rtd.css

```diff
@@ -176,6 +180,8 @@ blockquote {
   padding-bottom: 10px;
   z-index: 11;
   width: 100%;
+  font-weight: 600;
+  min-width: 1007px;
```

Comment:
if you have a small screen it would scroll 
![z2okwymzwh](https://cloud.githubusercontent.com/assets/13311684/18862120/62dbdef0-848a-11e6-86d7-097690f14477.gif)

i think it a standard behaviour for mobile..
![flqflkqddp](https://cloud.githubusercontent.com/assets/13311684/18862208/d1d62950-848a-11e6-85bb-b46ac34d00e6.gif)